src/tableproperties.js

```diff
@@ -36,3 +36,45 @@ export default class TableProperties extends Plugin {
 		return [ TablePropertiesEditing, TablePropertiesUI ];
 	}
 }
+
+/**
+ * A configuration of the table properties user interface (balloon). It allows to define:
+ *
+ * * the color palette for the table border color style field (`tableProperties.border.colors`),
+ * * the color palette for the table background style field (`tableProperties.backgroundColors`).
+ *
+ *		const tableConfig = {
+ *			tableProperties: {
+ *				border: {
+ *					colors: [
```

Comment:
Same thing about config namespace.

Reply:
I left a comment in https://github.com/ckeditor/ckeditor5/issues/6106. I think it's an unnecessary complication.

Reply:
See https://github.com/ckeditor/ckeditor5-table/pull/244#discussion_r378725172